plugins/inputs/snmp_heartbeat/snmp_heartbeat.go

```diff
@@ -0,0 +1,1072 @@
+package snmp_heartbeat
```

Comment:
We need to have README.md file in snmp_heartbeat directory.
This helps in users getting documentation on how to use the plugin.

Reply:
Added.